apps/site/src/content/events/digital-real-estate-summit.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Digital Real Estate Summit
+created-on: 2025-01-08T21:09:00.000Z
+updated-on: 2025-01-08T21:09:00.000Z
+published-on: 2025-01-08T21:09:00.000Z
+category: supported-sponsored
+location:
+  primary: "Miami, Florida"
+  region: north-america
+start-date: 2025-02-11T09:00:00.000Z
+end-date: 2025-02-11T18:00:00.000Z
+external-link: https://lu.ma/n4c8aotx?tk=GRGiWo
+seo:
+  twitter:
+    card: summary
+  description: This summit at the Frost Museum of Science, hosted by Magma, Propy,
+    Akila, Filecoin, and SustainaCities, highlights how digital assets empower
+    stakeholders in building a sustainable urban future from design to
+    operations.
```

Comment:
@oliviasigler - This SEO description is too large. Character count has to stay under 220 characters. And this is 377 characters long.

Reply:
I have adapted it so that we can merge it - Please make change it if you don't agree with it!